apps/frontend/src/components/layout/header.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { GeistSans } from 'geist/font/sans';
+import Link from 'next/link';
+
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar';
+import { Button } from '@/components/ui/button';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from '@/components/ui/dropdown-menu';
+
+export function Header() {
+  const isLoggedIn = true;
+  const user = {
+    name: 'John Doe',
+    email: 'john@example.com',
+    image: '/placeholder.svg',
+  };
+
+  return (
+    <header className='px-4 py-3 flex items-center justify-between'>
+      <Link href='/apps/frontend/public' className='flex items-center space-x-2'>
+        <span className={`${GeistSans.className} text-2xl font-bold tracking-tighter`}>MMMK</span>
+      </Link>
+
+      {isLoggedIn ? (
+        <DropdownMenu>
+          <DropdownMenuTrigger asChild>
+            <Button variant='ghost' className='relative h-8 w-8 rounded-full'>
+              <Avatar className='h-8 w-8'>
+                <AvatarImage src={user.image} alt={user.name} />
```

Comment:
We probably wont have user images, so this could stay placeholder constantly

Reply:
We agreed on the ability to set profile pictures.

Reply:
The ability for bands to upload pictures, no? Where would the average users picture be used?